corehq/motech/tests/test_value_source.py

```diff
@@ -634,3 +654,32 @@ def test_set_external_data(self):
 def test_doctests():
     results = doctest.testmod(corehq.motech.value_source, optionflags=doctest.ELLIPSIS)
     assert results.failed == 0
+
+
+def _create_case(case, utc_datetime) -> CommCareCaseSQL:
+    form = XFormInstanceSQL(
+        form_id=uuid4().hex,
+        xmlns='http://example.com/motech/test-value-source',
+        received_on=utc_datetime,
+        user_id=case.owner_id,
+        domain=case.domain,
+    )
+    transaction = CaseTransaction(
+        type=CaseTransaction.TYPE_FORM,
+        form_id=uuid4().hex,
```

Comment:
nit: Would it make more sense to have this reference the form above?
```suggestion
        form_id=form.form_id,
```